src/mon/OSDMonitor.cc

```diff
@@ -6052,6 +6086,10 @@ int OSDMonitor::prepare_command_pool_set(map<string,cmd_vartype> &cmdmap,
       ss << "pool size must be between 1 and 10";
       return -EINVAL;
     }
+    int r = check_pg_num(-1, p.get_pg_num(), n, &ss);
```

Comment:
pool instead of -1

Reply:
oh it doesn't affect the result though, nvm

Reply:
oh yes it does, since we pass the new size in

Reply:
fixed